src/pages/documentation/adobe-campaign-classic/release-notes.md

```diff
@@ -6,6 +6,8 @@
 
 * Major version update for [Adobe Campaign Classic workflows](./index.md) for Adobe Experience Platform Mobile SDKs on Android compatible with Mobile Core 2.0.0. This library is now available as an [open source project on GitHub](https://github.com/adobe/aepsdk-campaignclassic-android).
 
+Please note that the `registerDevice` API, similar to iOS, no longer provides a callback method to provide boolean registration status since a `false` value cannot be accurately used as a signal to retry request for success.
```

Comment:
... a callback method for registration status, since `false` values ... (same for the general notes)